feature/LP-134-OutlinedTextFieldInput

AndresIvanMedinaH opened the pull request:
**• Task’s name**

Fix: LpOutlinedTextFieldNumber needs to refactor / Re-name

**• Task’s link to navigate to jira**

https://devaptivist.atlassian.net/browse/LP-134?atlOrigin=eyJpIjoiMzQ5OTUzZGQwN2VkNDM4ZWJkZjYxZmJkMjdlZDIxMjAiLCJwIjoiaiJ9

**• General task’s description of what was added, deleted or updated in the code.**

• To receive alphanumeric values, since it could be used for any type of fieldf for generic purposes
• To have “enabled” option as parameter passed to function (“true” as the default value), so the user will be unable to edit/modify the text
• To change the name of LpOutlinedTextFieldNumber to LpOutlinedTextFieldInput for a general one of any text field

**• Attachment file of how looks your code in the app or in the preview if it is necessary.** 

![unit](https://user-images.githubusercontent.com/128720218/233160671-4ff0ee77-d9d8-4c82-8b64-fd44438ba0fa.jpg)
![unit2](https://user-images.githubusercontent.com/128720218/233160673-e4d51b0b-e31c-4fce-9295-3d8b69f7c8c0.jpg)

AndresIvanMedinaH committed:
feature/LP-134-OutlinedTextFieldInput

Changed parameters and added new ones
Enabled = false
Keyboard to accept alphanumeric
Changed name to some more generic

AndresIvanMedinaH committed:
feature/LP-134-OutlinedTextFieldInput

Changed parameters and added new ones
Enabled = false
Keyboard to accept alphanumeric
Changed name to some more generic

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:
```diff
@@ -147,8 +147,9 @@ fun LpOutlinedTextFieldMail(
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun LpOutlinedTextFieldNumber(
```
Please reformat the code

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:
```diff
@@ -147,8 +147,9 @@ fun LpOutlinedTextFieldMail(
-fun LpOutlinedTextFieldNumber(
+fun LpOutlinedTextFieldInput(
     modifier: Modifier,
+    enabled:Boolean = false,
```
Please don't forget to review the task's acceptance criteria. (second point)

AndresIvanMedinaH replied:
Considered and fixed

AndresIvanMedinaH committed:
Merge remote-tracking branch 'Irving/feature/LP-134-LPOutlineTextFieldInput' into feature/LP-134-LPOutlineTextFieldInput

# Conflicts:
#	core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

AndresIvanMedinaH committed:
Updated:
Enabled = True as  default parameter
reformat code
removed unused imports

Merge updates in local branch:
feature/LP-134-LPOutlineTextFieldInput' into feature/LP-134-LPOutlineTextFieldInput

# Solved conflicts:
#	core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

AndresIvanMedinaH committed:
Merge remote-tracking branch 'Irving/feature/LP-134-LPOutlineTextFieldInput' into feature/LP-134-LPOutlineTextFieldInput

#Solving space diff Conflicts:
#	core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

AndresIvanMedinaH committed:
Merge remote-tracking branch 'Irving/feature/LP-134-LPOutlineTextFieldInput' into feature/LP-134-LPOutlineTextFieldInput

# Conflicts:
#	core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

AndresIvanMedinaH committed:
Merge remote-tracking branch 'Irving/feature/LP-134-LPOutlineTextFieldInput' into feature/LP-134-LPOutlineTextFieldInput

# Conflicts:
#	core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt
